src/components/notifications/Keep.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <div class="mb-3">
+        <label for="webhook-url" class="form-label">{{ $t("Host URL") }}</label>
+        <input
+            id="webhook-url"
+            v-model="$parent.notification.webhookURL"
+            type="url"
+            pattern="https?://.+"
+            class="form-control"
+            required
+        />
+    </div>
+
+    <div class="mb-3">
+        <label for="webhook-apikey" class="form-label">{{ $t("API Key") }}</label>
+        <input
+            id="webhook-apikey"
+            v-model="$parent.notification.webhookAPIKey"
+            type="text"
+            class="form-control"
+            required
+        />
+    </div>
```

Comment:
At the end of the form we normally try to link to the relevant part of the docs so users are not lost.

Could you add such a link here as well?

Reply:
Added the docs link for better UX, link will be live once my PR is merged in Keep